polaris.shopify.com/src/styles/globals.scss

```diff
@@ -405,3 +405,15 @@ button {
     margin: 0 !important;
   }
 }
+
+.visuallyHidden {
```

Comment:
adding a global visually hidden style so that we can use it wherever, in this case so that we can have a hidden title for the component examples section